build/pr-validation.yaml

```
@@ -3,33 +3,43 @@
 trigger:
   branches:
     include:
-      - future
-      - master
+      - main
       - release/*
 
 pr:
   autoCancel: true # indicates whether additional pushes to a PR should cancel in-progress runs for the same PR. Defaults to true
   branches:
-    include: [ future, master, release/* ] # branch names which will trigger a build
+    include:  # branch names which will trigger a build
+    - main
+    - release/*
 
 name: $(Year:yyyy).$(Month).$(DayOfMonth)$(Rev:.r)
 
-jobs:
+variables:
+  NUGET_XMLDOC_MODE: none
+  NUGET_CREDENTIALPROVIDER_SESSIONTOKENCACHE_ENABLED: 'true'
+  packageFeed: 'https://pkgs.dev.azure.com/msazure/_packaging/ApiManagement/nuget/v3/index.json'
```

We are duplicating this for multiple pipelines, so let's re-use it!

Create `variables/nuget.yml` under this folder which has this:
```yaml
variables:
  NUGET_XMLDOC_MODE: none
  NUGET_CREDENTIALPROVIDER_SESSIONTOKENCACHE_ENABLED: 'true'
  packageFeed: 'https://pkgs.dev.azure.com/msazure/_packaging/ApiManagement/nuget/v3/index.json'
```

And then refer to it in every pipeline:
```suggestion
variables:
- template variables/nuget.yml
```

---

Will do in seperate PR